packages/components/src/Icon/Icon.tsx

```diff
@@ -47,7 +58,7 @@ export function Icon({ name, color, customColor, size = "base" }: IconProps) {
       xmlns="http://www.w3.org/2000/svg"
       viewBox={viewBox}
       className={svgClassNames}
-      data-testid={name}
+      data-testid={testID || name}
```

Comment:
There are two external instances where we are using `name` in jest tests. I personally do not see the harm in continuing to support that - but I'd also be happy to remove `name` from this `data-testid` and update the tests in Jobber if we think that's best.

Reply:
Thinking long term and about trying to implement this prop predictably across all of our library components, I am inclined to fix those JO tests and not support the `name` property here also being the testID. We won't always have a `name` prop on other components, so it wouldn't be a "standard" behavior and might get confusing.

Reply:
Agree with Jeff. If there ends up being a tiny bit of cleanup in JO for these changes, I'm okay with that. It's better for everything in the long-run to be consistent.

If it ever turns out to be a HUGE change though, we can talk about potentially pushing some of that work on teams (especially if we can accommodate in the short-term with a migration-friendly build for a while)

Reply:
Perfect! There will be cleanup in mobile for that too (because the same thing is happening there for this component) but I can pre-emptively make sure I have a draft PR up making those changes so we have a fast follow after merging into Atlantis.

I'll make the change

Reply:
ACTUALLY 👀 I found a good reason to keep `data-testid={testID || name}`.

In components where we have `suffix`, `prefix` or components like `Button` where we have `icon` as a prop:

```
interface ButtonFoundationProps {
 ...
  readonly icon?: IconNames;
```

We're bringing in `IconNames` and teams are using _that_ to test that an `Icon` is being used in these components as prefixes, etc.  So, only having `testID` here is not only more of a breaking change than I thought, I don't think we should expect consumers of `Button` to put a `testID` on a nested `Icon` that's being rendered as a prop.